pydantic_ai_slim/pydantic_ai/profiles/anthropic.py

```diff
@@ -1,8 +1,51 @@
 from __future__ import annotations as _annotations
 
+import importlib
+from collections.abc import Callable
+from dataclasses import dataclass
+from typing import Any, cast
+
+from .._json_schema import JsonSchema, JsonSchemaTransformer
 from . import ModelProfile
 
+TransformSchemaFunc = Callable[[Any], JsonSchema]
+
+try:  # pragma: no cover
+    _anthropic_module = importlib.import_module('anthropic')
+except Exception:
+    _anthropic_transform_schema: TransformSchemaFunc | None = None
+else:
+    _anthropic_transform_schema = cast(TransformSchemaFunc | None, getattr(_anthropic_module, 'transform_schema', None))
+
+
+@dataclass(init=False)
+class AnthropicJsonSchemaTransformer(JsonSchemaTransformer):
+    """Transforms schemas to the subset supported by Anthropic structured outputs."""
+
+    def walk(self) -> JsonSchema:
+        schema = super().walk()
+        helper = _anthropic_transform_schema
+        if helper is None:
+            return schema
+        try:  # pragma: no branch
+            # helper may raise if schema already transformed
```

Comment:
Really? Why? What kind of error?

Reply:
I honestly wasn't sure how we handle like backwards compatibility stuff, that was the reason for the weird import and the many conditionals, this is much simpler now with the sdk